packages/hierarchies-react/README.md

```diff
@@ -242,3 +242,70 @@ function MyTreeComponentInternal({ imodelAccess, imodelKey }: { imodelAccess: IM
   return <TreeRenderer rootNodes={rootNodes} {...state} />;
 }
 ```
+
+## Localization
+
+Localization can be enabled for `TreeRenderer` component and `useTree` and `useUnifiedSelectionTree` hooks by providing an object with localized strings that will be used instead of the default english ones.
+
+Example:
+
+```tsx
+const localizedStrings = {
+  loading: "Loading...",
+  filterHierarchyLevel: "Apply filter",
+  clearHierarchyLevelFilter: "Clear active filter",
+  noFilteredChildren: "No child nodes match current filter",
+  resultLimitExceeded: "There are more items than allowed limit of {{limit}}.",
+  resultLimitExceededWithFiltering: "Please provide <link>additional filtering</link> - there are more items than allowed limit of {{limit}}.",
+  increaseHierarchyLimit: "<link>Increase the hierarchy level size limit to {{limit}}.</link>",
+  increaseHierarchyLimitWithFiltering: "Or, <link>increase the hierarchy level size limit to {{limit}}.</link>",
+  unspecified: "Unspecified",
+  other: "Other",
```

Comment:
These two used by `useTree` should be separated from the other ones. `useTree` can be used without `TreeRenderer` so in that case it does not make sense to require all these additional strings that are used only by renderer.

Reply:
They are separate, I just merged the object into one in the `README`. Split into two to be more clear.

Reply:
I personally liked the non-split version better. Maybe adding a couple of comments would've been a better change?
```ts
const localizedStrings = {
  // strings for the `useTree` hook
  ...

  // strings for `TreeRenderer` and `TreeNodeRenderer`
  ...
};
```

Reply:
Reverted the change and left comments.